system-test/testTypes.ts

```diff
@@ -46,3 +47,23 @@ export interface Test {
   row_keys_read: {};
   createReadStream_options: GetRowsOptions;
 }
+
+interface CreateReadStreamResponse {
+  row_keys?: string[];
+  end_with_error?: 4;
+}
+
+interface CreateReadStreamRequest {
+  rowKeys: string[];
+  rowRanges: google.bigtable.v2.IRowRange[];
+  rowsLimit?: number;
+}
+export interface ReadRowsTest {
+  createReadStream_options?: GetRowsOptions;
+  max_retries: number;
+  responses: CreateReadStreamResponse[];
+  request_options: CreateReadStreamRequest[];
+  error: number;
+  row_keys_read: string[][];
+  name: string;
+}
```

Comment:
In here there is an interesting mix of snake_case and camelCase - is there a reason why both have to be used? Or can they all be migrated to a more typical JS-y camelCase?

Reply:
They can be migrated, but it means that the json file containing all the test data in https://github.com/googleapis/nodejs-bigtable/blob/3f5db964062502cd70235714eb7b5fb62a211d4f/system-test/data/read-rows-retry-test.json#L112 needs to be updated to match.